docs/user-guide/admin-client/consumer-group.rst

```diff
@@ -0,0 +1,201 @@
+This call updates metadata about the consumer group itself (not the members).
+
+::
+
+  $ pulp-admin rpm consumer group update --group-id test-group --display-name new-name
+
+  Consumer Group [test-group] successfully updated
+
+The new consumer group can be seen with its unique ID and display name.
+
+::
+  $ pulp-admin rpm consumer group list
+
+  +----------------------------------------------------------------------+
+                              Consumer Groups
+  +----------------------------------------------------------------------+
+
+  Id:           test-group
+  Display Name: new-name
+  Description:  None
+  Consumer Ids: 
+  Notes: 
+
+Group can be removed from the Pulp server using the delete command.
+Deleting a consumer group has no effect on the consumers that are members of the group.
+
+::
+
+  pulp-admin rpm consumer group delete --group-id test-group
+
+  Consumer Group [test-group] successfully deleted
+
+
+Group membership
+----------------
+
+Consumers can be associated and unassociated with any existing consumer group.
+
+::
+  $ pulp-admin rpm consumer group members
+
+  Usage: pulp-admin [SUB_SECTION, ..] COMMAND
+  Description: manage members of repository groups
+
+  Available Commands:
+    add    - add consumers to an existing group
+    list   - list the consumers in a particular group
+    remove - remove consumers from a group
+
+The call is idempotent if the consumer is already memeber of the group.
+
+::
+  $ pulp-admin rpm consumer group members add --group-id test-group --str-eq='id=consumer1'
+
+  Consumer Group [test-group] membership updated
+
+  $ pulp-admin rpm consumer group list 
+
+  +----------------------------------------------------------------------+
+                              Consumer Groups
+  +----------------------------------------------------------------------+
+
+  Id:           test-group
+  Display Name: None
+  Description:  None
+  Consumer Ids: consumer1
+  Notes:    
+
+  $ pulp-admin rpm consumer group members list --group-id test-group
+
+  +----------------------------------------------------------------------+
+                           Consumer Group Members
+  +----------------------------------------------------------------------+
+
+  Id:           consumer1
+  Display Name: consumer1
+  Description:  None
+  Notes:   
+
+Now you can see the consumer1 is part of the group.
+Same logic applies to removal of consumers from the group.
+
+::
+  $ pulp-admin rpm consumer group members remove --group-id test-group --str-eq='id=consumer1'
+
+  Consumer Group [test-group] membership updated
+
+
+Repository Binding
+------------------
+
+``bind`` command allows to bind each consumer in a consumer group to a repository.
+
+::
+  $ pulp-admin rpm consumer group bind --consumer-group-id test-group --repo-id zoo
+
+  Consumer Group [test-group] successfully bound to repository [zoo]
+
+Same for ``unbind`` - unbinds each consumer in a consumer group from a repository.
```

Comment:
How about "You can also use the ``unbind`` comand to unbind each consumer in a…"?